src/sql/tables.sql

```diff
@@ -92,7 +93,8 @@ CREATE TABLE msgs (
 -- deleted. It is convenient to store it here because UI
 -- needs this value to display how much time is left until
 -- the message is deleted.
-    ephemeral_timestamp INTEGER DEFAULT 0
+    ephemeral_timestamp INTEGER DEFAULT 0,
+    encryption_modus INTEGER DEFAULT 0
```

Comment:
I just went ahead and implemented it, don't know the core codebase too well. So if you tell me that it would make more sense somewhere else i'm happy to change this :)

Reply:
The other reason i put it is that the encryption modus is tied to the message in case it gets sent later/resent. Not sure if this is really a feature or what users would expect.